utils/mock_qpu/quantinuum/__init__.py

```diff
@@ -5,11 +5,11 @@
 # This source code and the accompanying materials are made available under     #
 # the terms of the Apache License 2.0 which accompanies this distribution.     #
 # ============================================================================ #
-import cudaq
 
-from fastapi import FastAPI, Request, HTTPException, Header
-from typing import Optional, Union
-import uvicorn, uuid, json, base64, ctypes
+import cudaq
```

Comment:
Doesn't need changing in this script but [relevant issue](https://github.com/NVIDIA/cuda-quantum/issues/408)